<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="login_top_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>428</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/login/img/login/top.png);</string>
   </property>
   <widget class="QPushButton" name="btn_small">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>8</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>5</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="topic">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>320</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 40pt &quot;华文行楷&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>标注平台</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="login_con_widget" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>181</y>
     <width>311</width>
     <height>95</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>用户ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_admin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>76</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>管理员</string>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_test">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>76</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>审核员</string>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_register">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>注册账号</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pass">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>找回密码</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="user_img">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
